package.json

```diff
@@ -22,6 +22,7 @@
     "merge-stream": "0.1.6",
     "rimraf": "2.2.8",
     "run-sequence": "1.0.1",
+    "stylus": "~0",
```

Comment:
This was previously being set in marketplace-gulp. I'm not sure if that's a better place for it but it seemed weird to have to update that package to update stylus in this one.